constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import API from "../utils/api.js";
+import moment from "moment";
+
+export class AFHPage extends React.Component {
+    state = {
+        currentTab: "Math",
```

Comment:
rename this currentSubject. This value is ALWAYS one of (math, english, or computer).
We will use it as a key to figure out which subject we are dealing with atm